input-output-hk/lace
# Feat/LW-7128 add handle update screen

# Checklist

- [x] LW-7128

---

## Proposed solution

Adds the flow for the warning of the handles that have changed ownership.
Adds a delay on retrying because when a user has several handles, and renavigates, koralabs will fail because of the limit on the number of requests to the API. 
Adds the comparison in between stake keys instead of addresses. 

## Testing
 - Try to add a handle in the popup and browser 
 - Try to add addresses in both 
 - Send a minted handle to another address 
 - Click in the warning and either delete or update the contact 
 - Check the warning disappears when the handles have been minted although if you have more than 5 handles, you might need to wait 5 seconds to see the warning. 

## Screenshots

<img width="1534" alt="Screenshot 2023-07-13 at 10 28 43" src="https://github.com/input-output-hk/lace/assets/4052063/83b8d253-7d4c-44d5-8126-012a64c769c1">
<img width="1548" alt="Screenshot 2023-07-13 at 10 28 55" src="https://github.com/input-output-hk/lace/assets/4052063/98f623da-2f1d-40a6-9eb8-4725e9d88578">
<img width="1551" alt="Screenshot 2023-07-13 at 10 29 06" src="https://github.com/input-output-hk/lace/assets/4052063/0b393252-8f1d-4250-8656-68d3e429b022">
<img width="1566" alt="Screenshot 2023-07-13 at 10 29 22" src="https://github.com/input-output-hk/lace/assets/4052063/e19607a1-80f0-432b-a09d-16f9c7691eeb">



---
# Allure report
`allure-report-publisher` generated test report!



**smokeTests**: ✅ [test report](https://lace-qa:8PP5wBaDV6UoXj@d1terqjryk7mu9.cloudfront.net/linux/chrome/1085/5614281440/index.html) for [e9475637](https://github.com/input-output-hk/lace/pull/262/commits/e94756372e5ef21d6558b6b5c8615909502face9)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 35     | 0      | 0       | 0     | 35    | ✅     |
